Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -1026,6 +989,41 @@ class CombinatorialKalmanFilter {
       }
     }
 
+    void storeLastActiveTip(result_type& result) const {
+      const auto& [currentTip, tipState] = result.activeTips.back();
+      // @TODO: Keep information on tip state around so we don't have to
+      //        recalculate it later
```

Comment:
in a future PR?

Reply:
I copied this TODO over from the previous version. I don't plan on following up on this for now. Would be nice to have but at the same time will not really have an impact on performance.